calico-cloud_versioned_docs/version-3.16/image-assurance/understanding-scan-results.mdx

```diff
@@ -118,8 +104,33 @@ Images with no associated CVEs are not included in the exported data (regardless
 
 :::
 
+### Running Images tab
+
+This tab shows CVEs associated with running pods in your cluster so you can assess pod vulnerability and remediate.For example, you can see image exposure by viewing the  **Clusters** and **Running Instances** columns that show the number of running instances in clusters that are connected to {{prodname}}.
+
+### Troubleshooting 
+
+#### No results in the Running Images tab
+
+- The Image Assurance may not be enabled and running
+- If you integrated the Image Assurance scanner into your build pipeline, and you are not using the default you may containerd socket path (`/run/containerd/containerd.sock`), image information won't be available. To update the CRI socket path for a cluster, run the following command:
```

Comment:
This sentence looks grammatically incorrect, but also factually incorrect. 

In terms of grammar, this part looks wrong "... and you are not using the default you may containerd socket path ...". 

In terms of factual information, the CRI path socket path is for the CR Adaptor (and also in-cluster scanner), which runs in the customer's managed cluster, in order to collect pod metadata to figure out which images are running. 

The Image Assurance scanner (I'm assuming the CLI scanner here due to the context of the sentence) in your build pipeline has nothing to do with the CLI socket path configuration. 

Also, the part that says "... image information won't be available." should say runtime information instead of images.

Reply:
Changed to: If you have modified the default containerd socket path (`/run/containerd/containerd.sock`), runtime image information will not be available. To update the CRI socket path for a cluster, run the following command: